__tests__/integration.test.js

```diff
@@ -285,3 +285,28 @@ describe("PATCH: /api/articles/:article_id", () => {
         });
     });
 });
+describe("DELETE: /api/comments/:comment_id", () => {
+  test("DELETE: responds 204 status code when successfully deletes the specified comment based on comment_id", () => {
+    return request(app)
+    .delete("/api/comments/16")
+    .expect(204)
+  })
```

Comment:
You might want to check that this also returns no content ie -  **expect(body).toEqual({})**